core/trino-main/src/main/java/io/trino/operator/PagesSpatialIndexSupplier.java

```diff
@@ -41,6 +41,7 @@
 
 import static com.google.common.base.Verify.verifyNotNull;
 import static io.airlift.slice.SizeOf.instanceSize;
+import static io.trino.SystemSessionProperties.getSpatialGeometryAccelerationDegree;
```

Comment:
Can we extract `TestSpatialJoinOperator` to run the tests with all possible acceleration degree ?